EXAMPLES/MATRIX_MARKET/Makefile.am

```diff
@@ -6,7 +6,6 @@ EXTRA_DIST = README
 check_PROGRAMS = arpackmm
 dist_check_SCRIPTS = arpackmm.sh
 dist_check_DATA = An.mtx As.mtx Az.mtx B.mtx Bz.mtx
-SH_LOG_COMPILER = bash
 TESTS = arpackmm.sh
```

Comment:
@yurivict : wasn't this needed on FreeBSD ? Does `./configure --enable-icb-exmm` runs OK on FreeBSD ?